src/actions/user/signupAction.js

```diff
@@ -0,0 +1,27 @@
+import httpClient from '../../helpers/index';
+import {
+  USER_SIGNUP_REQUEST,
```

Comment:
`SIGNUP_*` is sufficient. Pls lets stick with a convention.

Reply:
@dinobi I have resolved this.